airiam/find_unused/RuntimeIamScanner.py

```diff
@@ -12,6 +14,16 @@
 ERASE_LINE = '\x1b[2K'
 
 
+def get_iam_data_file(account_id=None):
+    if account_id is None:
+        return IAM_DATA_FILE_NAME
+    else:
+        parent = f"./aircache/{account_id}"
+        pathlib.Path(parent).mkdir(parents=True, exist_ok=True)
+        p = f"{parent}/{IAM_DATA_FILE_NAME}"
+        return p
```

Comment:
@manishsingh10895 How about this?
```suggestion
def get_iam_data_file(account_id: str):
    parent = f"./aircache/{account_id}"
    pathlib.Path(parent).mkdir(parents=True, exist_ok=True)
    p = f"{parent}/{IAM_DATA_FILE_NAME}"
    return p
```

Reply:
Thanks, exactly what I thought.

Reply:
@nimrodkor I didn't realize there was a commit suggestion button here. I made changes and made a new commit